index.html

```diff
@@ -139,6 +140,13 @@ <h2>Methods</h2>
         </dd>
       </dl>
     </section>
+    <section>
+      <h2>Modifications to existing procedures</h2>
+
+      <p>In the <a data-cite="WEBRTC#dom-peerconnection">Constructor</a> algorithm,
+      add a step right before step 2 saying "Run [[WEBAPPSEC-CSP#should-block-rtc-connection]]
+      on the current global object. Throw a {{NotAllowedError}} if it returns `Blocked`."</p>
```

Comment:
To avoid referencing step numbers, we've been recommending the [following pattern](https://w3c.github.io/webrtc-identity/#creating-an-offer) (it might limit us to adding to the top and bottom of algorithms, but we'll see how it goes):

```suggestion
      <p>Whenever the <a data-cite="WEBRTC#dom-peerconnection">RTCPeerConnection constructor</a>
      algorithm is invoked, run the following steps instead:
        <ol>
          <li>If [[WEBAPPSEC-CSP#should-block-rtc-connection]] on the current global object returns "Blocked",
          throw a {{NotAllowedError}}."</li>
          <li>Return the result of running the original
          <a data-cite="WEBRTC#dom-peerconnection">RTCPeerConnection constructor</a> algorithm.</li>
        </ol>
      </p>
```

Also, using `[[` `]]` appears to not work well for some reason, or is that a limitation of Preview?

Reply:
It works well enough in this case, done. Re: `[[` `]]`, it could be that the name of the doc is wrong (it was a shot in the dark, I'm not sure how to locate the correct name). Or it could be something else; as I said in the original post I don't totally grok how all of the formatting for these docs works.